### DISCUSSION TITLE
messages are delated

### DISCUSSION THREAD

**Discussion by artavenue34** [community/community#82988] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

hi my webite related to[ round rungs](https://www.artavenue.com/) but the problem is that the my conversation with the users are deleted so that I face many issues please help me in this regard to recover all my conversation .

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Akash1134** [comment]
Hi @artavenue34 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
